infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/query/parsing/impl/InfinispanRemoteBasedQueryParserService.java

```diff
@@ -34,7 +34,7 @@
 
 	@Override
 	public QueryParsingResult parseQuery(SessionFactoryImplementor sessionFactory, String queryString, Map<String, Object> namedParameters) {
-		throw new UnsupportedOperationException( "The Infinispan Remote query parser supports parameterized queries" );
+		throw new UnsupportedOperationException( "The Infinispan Remote query parser doesn't support parameterized queries" );
```

Comment:
I think that supports was right. Like in **Neo4jBasedQueryParserService**.
In a nutshell: the client driver supports parameters natively, like in Neo4j, unlike MongoDB. Thus, from what I understood, other method must be called, because the passer supports parameterized queries.

Reply:
> Thank you @DavideD, reducing the amount of regular expressions needed for the parser was very good. I have just a minor doubt about the 5642f87.

Thanks @fax4ever,. Got it. Could you remove that commit and merge everything else, please? The message seems misleading but that's an issue for another time.

Reply:
Sure!